packages/ansible_base.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Ansible_base < Package
+  description 'Ansible base libraries and functions.'
+  homepage 'https://www.ansible.com/'
+  version '2.10.9'
+  license 'GPL-3'
+  compatibility 'all'
+  source_url 'https://files.pythonhosted.org/packages/2f/7c/36119f0695fe48fd5eeb33a78c11137642abf1f67b477f31b9772da5a85d/ansible-base-2.10.9.tar.gz'
```

Comment:
Can we use the GitHub release source instead?
https://github.com/ansible/ansible/archive/v2.10.9.tar.gz
It looks like an update is available also:
https://github.com/ansible/ansible/archive/v2.11.0.tar.gz

Reply:
No, because this github source combines ansible and ansible-base.

Reply:
What is wrong with that?  I guess I'm confused why we would want/need separate packages?

Reply:
chromebooks have notoriously small hard drives, and also pip separates them. some packages require ansible-base and not ansible